roxie/ccd/ccdmain.cpp

```diff
@@ -591,6 +592,8 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
             argv[i] = "--csv";
         else if (strsame(argv[i], "-raw"))
             argv[i] = "--raw";
+        else if (strsame(argv[i], "-v"))
```

Comment:
You could argue that this functionality should go into the config code itself - I didn't, because logging has not been set up at the time that is called (as logging depends on the settings...).